<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>1051</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>211</width>
      <height>273</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>                           Radio</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>   Pos X       </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>           Pos Y   </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="posX">
           <property name="minimum">
            <double>-99999.990000000005239</double>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="posY">
           <property name="minimum">
            <double>-99999.990000000005239</double>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>            Vel Y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="velY">
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.990000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>   Vel X</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="velX">
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.990000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="verificar">
         <property name="text">
          <string>verificar pos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>                    Radio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="radio">
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>                       Masa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="masa">
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>360</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ingresar">
       <property name="text">
        <string>Ingresar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="simular">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>460</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>simular</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
